types/dropbox_types.d.ts

```diff
@@ -6315,11 +6316,11 @@
        */
       member: MemberSelector;
       /**
-       * Defaults to TagRef(Union('AccessLevel', [UnionField('owner', Void,
-       * False, None), UnionField('editor', Void, False, None),
-       * UnionField('viewer', Void, False, None),
-       * UnionField('viewer_no_comment', Void, False, None), UnionField('other',
-       * Void, True, None)]), 'viewer').
+       * Defaults to TagRef(Union(u'AccessLevel', [UnionField(u'owner', Void,
```

Comment:
All fo these u'' should be gone; rerun the generator using python 3 to avoid this